docs/settings/security-settings.asciidoc

```diff
@@ -162,6 +162,15 @@ Adds a message accessible at the login UI with additional help information for t
 xpack.security.authc.selector.enabled {ess-icon}::
 Determines if the login selector UI should be enabled. By default, this setting is set to `true` if more than one authentication provider is configured.
 
+[float]
+[[authentication-access-agreement-settings]]
+==== Configure a default access agreement
```

Comment:
Other sections include this as a lead-in sentence:

>You can configure the following settings in the `kibana.yml` file.

Reply:
Oo! I'll add that in, thanks!!